web-app/roomberryWrapper.php

```diff
@@ -1,242 +1,216 @@
 <?php
```

Comment:
There has been 420 line changes only in this commit while the whole roomberry.py has only 522 lines.
I can't even find where the Multimedia path is created.
Please be careful when proposing a commit/pull request. Change only the intended lines, otherwise it is impossible to track the changes.